dockerfiles/Dockerfile

```diff
@@ -198,6 +225,12 @@ COPY --from=deb-extractor /dpkg /
 # Copy certificates
 COPY --from=builder /etc/ssl/certs /etc/ssl/certs
 
+# Copy vsomeip
+COPY --from=builder-vsomeip /usr/local/lib/libvsomeip* /usr/local/lib/
+COPY --from=builder-vsomeip /usr/local/lib/cmake/vsomeip3/* /usr/local/lib/cmake/vsomeip3/
+COPY --from=builder-vsomeip /usr/local/lib/pkgconfig/vsomeip3.pc /usr/local/lib/pkgconfig/
+COPY --from=builder-vsomeip ./usr/local/include/vsomeip/. /usr/local/include/vsomeip/
+
```

Comment:
If we vendor the code in then it removes this brittle copying as the library should be packaged ideally statically or as a direct dependency, plus with all its dependencies identified. 

A `COPY` like this is brittle as it does not capture any transitive dependencies and potentially breaks at runtime (not build time or test time) for production images if anything changes in the future.